src/manifolds/SegreWarpedMetric.jl

```diff
@@ -0,0 +1,301 @@
+@doc raw"""
+    WarpedMetric{A} <: AbstractMetric
```

Comment:
Can we maybe
* provide a bit of information, what `A` is? Always just a number?
* provide a nicer constructor (and a corresponding show method) `WarpedMetric(a::Real)` or so?

Reply:
Here is a suggestion for information on `A`:
```
``A = 1`` corresponds to the usual Segre manifold.
The Segre manifold is a cone in the sense that if ``p \in \mathcal{S}``, then ``a p \in \mathcal{S}`` for all ``r \neq 0``. The tangent subspace at ``p`` defined ``\mathrm{d} (r p) / \mathrm{d} r`` is called the _radial_ direction. ``A < 1`` puts less weight on the directions orthogonal to the radial direction compared to ``\mathcal{S}``, while ``A > 1`` puts more weight on those directions.
```

Reply:
>     * provide a nicer constructor (and a corresponding show method) `WarpedMetric(a::Real)` or so?

For sure!

Reply:
The doc string should mention the constructor, the rest looks very good.